level2/instance.tf

```diff
@@ -63,14 +64,14 @@ resource "aws_security_group" "public" {
   }
 }
 
-resource "aws_instance" "private_webserver" {
+resource "aws_instance" "webserver_2" {
   ami                  = data.aws_ami.amazonlinux2.id
```

Comment:
you don't have to create two instance of private instance. you can use count argument to create many instance of the instance